Src/Web/Web.Shared.Net/RequestTrackingTelemetryModule.cs

```diff
@@ -216,7 +241,10 @@ public void OnEndRequest(HttpContext context)
                 requestTelemetry.Source = telemetrySource;
             }
 
-            this.telemetryClient.TrackRequest(requestTelemetry);
+            if (this.childRequestTrackingSuppressionModule?.OnEndRequest_ShouldLog(context) ?? true)
+            {
+                this.telemetryClient.TrackRequest(requestTelemetry);
+            }
```

Comment:
Please add else {} block, and log that we (childRequestTrackingSuppressionModule) are dropping this request.